src/utils/schema-utils.js

```diff
@@ -257,7 +262,14 @@ export function getSampleValueByType(schemaObj) {
   if (typeValue.match(/^string/g)) {
     if (schemaObj.enum) { return schemaObj.enum[0]; }
     if (schemaObj.const) { return schemaObj.const; }
-    if (schemaObj.pattern) { return schemaObj.pattern; }
+    if (schemaObj.pattern) {
+      const examplePattern = schemaObj.pattern.replace(/[+*](?![^\][]*[\]])/g, '{8}').replace(/\{\d*,(\d+)?\}/g, '{8}');
```

Comment:
nit: this line would be unnecessary with randexp.js, as randexp will just put the minimum number of characters (1), which is cleaner than 8 IMO